mod-infra/src/main/scala/at/forsyte/apalache/infra/passes/options.scala

```diff
@@ -0,0 +1,28 @@
+package at.forsyte.apalache.infra.passes
+
+/** Collects the passoptions supported for configuring Apalache's various passes */
+object options {
+
+  /**
+   * Defines the SMT encoding options supported
+   *
+   * @author
+   *   Shon Feder
+   */
+  sealed abstract class SMTEncoding
+
+  object SMTEncoding {
+    final case object Oopsla19 extends SMTEncoding {
+      override def toString: String = "oopsla19"
+    }
+    final case object Arrays extends SMTEncoding {
+      override def toString: String = "arrays"
+    }
+
+    val ofString: String => SMTEncoding = {
+      case "arrays"        => Arrays
+      case "oopsla19"      => Oopsla19
```

Comment:
Having arrays capitalized is fine, but `Oopsla19` is a bit odd. Not sure what would be a good workaround though.

Reply:
Hmmm true, we have it in all caps in other places, e.g. `TestArenaWithOOPSLA19`.